exporters/metric/cortex/cortex.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/golang/snappy"
 	"github.com/prometheus/prometheus/prompb"
 
+	"go.opentelemetry.io/otel/sdk/resource"
+
```

Comment:
nit: can you remove this line and group the OTel imports together?

Reply:
No order in these files, so not sure it matters.

Reply:
it is just a "nice to have". as for me: feel free to ignore my comments and thanks a lot for your PR 🎉 😉

Reply:
I think we should enforce an order and fix all the files.

Reply:
Happy to do that in a separate PR.